src/Console/ServerBlock.php

```diff
@@ -48,14 +48,18 @@ protected function getHelp()
 		$help = <<<HELP
 console serverblock - Manage blocked server domain patterns
 Usage
-	bin/console serverblock [-h|--help|-?] [-v]
-	bin/console serverblock add <pattern> <reason> [-h|--help|-?] [-v]
-	bin/console serverblock remove <pattern> [-h|--help|-?] [-v]
+    bin/console serverblock [-h|--help|-?] [-v]
```

Comment:
Why did you change the indentation style?

Reply:
IIRC because it was in a mixed state and spaces in the help block made more sense to me then tabs.